package.json

```diff
@@ -135,7 +135,6 @@
     "redux-form": "7.0.2",
     "redux-thunk": "1.0.0",
     "rtlcss": "2.0.5",
-    "sanitize-html": "1.11.1",
```

Comment:
Have you considered placing the majority of this code into another npm package so a wider audience can benefit? (other people/companies, and even our other projects?)

Reply:
There already is a good package that solves the same usecase in a more general purpose way: https://github.com/cure53/DOMPurify.

This implementation is super specific which is why it was able to be so small. I don't think (at this point) that it is worth publishing separately

Reply:
Not really because it's overly specific to our needs here. As I brought up in response to something @samouri asked, this is in no way a generic system and that's a big reason why we're able to trim it up. I think that "another npm package" is exactly what `sanitize-html` was. We simply don't need all that functionality here.